modules/fs2/src/main/scala/io/janstenpickle/trace4cats/fs2/syntax/Fs2StreamSyntax.scala

```diff
@@ -18,26 +18,65 @@ trait Fs2StreamSyntax {
     def inject(ep: EntryPoint[F], name: String): TracedStream[F, A] =
       inject(ep, _ => name, SpanKind.Internal)
 
+    def trace(reader: ResourceKleisli[F, SpanParams, Span[F]], name: SpanName): TracedStream[F, A] =
```

Comment:
Here and below - is `reader` a good name for this param having the type renamed?

Reply:
hmm yeah, good point